Core/StringHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Core
+{
+    public static class StringHelper
+    {
+        public static int GetStableHashCode(this string str)
+        {
+            unchecked
+            {
+                var hash1 = 5381;
+                var hash2 = hash1;
+
+                for(var i = 0; i < str.Length && str[i] != '\0'; i += 2)
+                {
+                    hash1 = ((hash1 << 5) + hash1) ^ str[i];
+                    if (i == str.Length - 1 || str[i+1] == '\0')
+                        break;
+                    hash2 = ((hash2 << 5) + hash2) ^ str[i+1];
+                }
+
+                return hash1 + (hash2*1566083941);
```

Comment:
Codacy found an issue: [Add underscores to this numeric value for readability.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6077033)